package.json

```diff
@@ -40,8 +40,10 @@
   },
   "homepage": "https://github.com/eslint-community/eslint-plugin-eslint-plugin#readme",
   "dependencies": {
+    "@typescript-eslint/utils": "^6.20.0",
```

Comment:
https://github.com/eslint-community/eslint-plugin-eslint-plugin/actions/runs/7789482239/job/21241206507?pr=433:

```plaintext
> nyc --all --check-coverage --include lib mocha tests --recursive


/home/runner/work/eslint-plugin-eslint-plugin/eslint-plugin-eslint-plugin/node_modules/@typescript-eslint/scope-manager/dist/referencer/ClassVisitor.js:123
        withMethodDecorators ||=
                             ^^^

SyntaxError: Unexpected token '||='
    at wrapSafe (internal/modules/cjs/loader.js:102[9](https://github.com/eslint-community/eslint-plugin-eslint-plugin/actions/runs/7789482239/job/21241206507?pr=433#step:5:10):16)
```

Is this blocked on dropping Node 14 support then?

Reply:
Is there an older version of the dependency we can use that will still work for Node 14?

In terms of dropping for old Node/ESLint, I'd be open to doing this as part of a major version soon, although there's some work for that in #269.